egs/swbd/s5c/local/reverberate/run_ivector_common.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash 
+#set -e
+# this script is based on local/nnet3/run_ivector_common.sh
+# but it operates on corrupted training/dev/test data sets
+
+. cmd.sh
+
+stage=1
+snrs="20:10:15:5:0"
+num_data_reps=3
+ali_dir=exp/tri4_ali_nodup_sp
+
+set -e
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+mkdir -p exp/nnet3_reverberate
+if [ $stage -le 0 ]; then
```

Comment:
We will assume that the simulated RIRs are available on openslr.org, so just download from there.
We will have to write a separate script to prepare these RIRs and upload to openslr.org, but that need not be part of the main recipe scripts. It can be an supporting script in the same recipe folder.